site/data/en/skill.yml

```diff
@@ -8,26 +8,35 @@ skill:
   skill_item:
     # skill item loop
     - name: Angular
-      percent: 90%
+      percent: 95%
 
+    # skill item loop
     - name: React
+      percent: 90%
+
+    # skill item loop
+    - name: Vue.js
       percent: 80%
 
     # skill item loop
     - name: JavaScript / TypeScript
-      percent: 90%
+      percent: 95%
 
     # skill item loop
-    - name: .NET 8 / .NET Framework 4.8.1
```

Comment:
I think maybe because of the ordering, .NET versions are overwritten, please revert it back